scripts/microbe_stage/microbe_editor/microbe_editor.as

```diff
@@ -1185,6 +1201,31 @@ class MicrobeEditor{
                 GetThriveGame().playerMovedToPatch(targetPatch);
             }
 
+            // Change species name
+            if(newName.findFirst(" ") > -1){
+                playerSpecies.genus = newName.split(" ")[0];
+                playerSpecies.epithet = newName.split(" ")[1];
+                LOG_INFO("MicrobeEditor: Player species name is now " + playerSpecies.genus + " " + playerSpecies.epithet);
+            } else {
+                LOG_INFO("MicrobeEditor: Invalid newName: " + newName);
+            }
+
+            auto membraneComponent = world.GetComponent_MembraneComponent(player);
+            // Change species membrane
+            playerSpecies.speciesMembraneType = membrane;
+            LOG_INFO("MicrobeEditor: Player species membrane type is now " + int(playerSpecies.speciesMembraneType));
+            membraneComponent.setMembraneType(membrane);
+            // Change species membrane color
+            playerSpecies.colour = colour;
+            LOG_INFO("MicrobeEditor: Player species colour is now RGB: " + playerSpecies.colour.X + " " + playerSpecies.colour.Y + " " + playerSpecies.colour.Z);
+            membraneComponent.setColour(colour);
+            membraneComponent.clear();
+            MicrobeComponent@ microbeComponent = cast<MicrobeComponent>(world.GetScriptComponentHolder("MicrobeComponent").Find(player));
+
+            if(@microbeComponent.species != @playerSpecies)
```

Comment:
Handle comparison syntax that I prefer: `microbeComponent.species !is playerSpecies`